### DISCUSSION TITLE
[Question][Hw2] What is this operation?

### DISCUSSION THREAD

**Discussion by Jihwan-Yoon** [prosyslab-classroom/cs348-information-security#689] [Q&A]
Name: Jihwan Yoon

![image](https://user-images.githubusercontent.com/78306242/230005144-0f2c2063-e13a-4b21-9c5d-6da44254a0ec.png)

What is this X like operator? I thought it is multiplication, but I don't think so. 

It results 0011 X 0111 = 1001....

What is it?

**Comment by pingpingy1** [comment]
It's the multiplication operation as described in section 2.3. There is also the entire multiplication table in the appendix.

**Comment by Jihwan-Yoon** [comment]
Oh, I see
Thank you
